<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CadastroProfissional</class>
 <widget class="QDialog" name="CadastroProfissional">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Profissional</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Cadastrar prof.</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="labelNome">
           <property name="text">
            <string>Nome</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditNome">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 127);</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="labelEmail">
           <property name="text">
            <string>E-mail</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="lineEditEmail">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 127);</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4" rowspan="2">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Status</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QRadioButton" name="radioButtonAtivo">
              <property name="text">
               <string>Ativo</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButtonInativo">
              <property name="text">
               <string>Inativo</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelTel">
           <property name="text">
            <string>Telefone</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="LineEditTel" name="lineEditTel">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 127);</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="labelTipo">
           <property name="text">
            <string>Tipo</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QComboBox" name="comboBoxTipo">
           <item>
            <property name="text">
             <string>ARQUITETO</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ENGENHEIRO</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DESIGNER</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>OUTRO</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0" colspan="5">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Dados bancários:</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="2">
               <widget class="QLabel" name="labelCP">
                <property name="text">
                 <string>CP:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="labelBanco">
                <property name="text">
                 <string>Banco:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="labelAgencia">
                <property name="text">
                 <string>Agência:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="labelCC">
                <property name="text">
                 <string>CC:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEditAgencia"/>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="lineEditCC"/>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEditBanco"/>
              </item>
              <item row="0" column="3">
               <widget class="QLineEdit" name="lineEditCP"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonCadastrar">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Cadastrar</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAtualizar">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Atualizar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonBuscar">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/search.png</normaloff>:/search.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonNovoCad">
       <property name="text">
        <string>Novo Cadastro</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRemover">
       <property name="text">
        <string>Remover</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEditTel</class>
   <extends>QLineEdit</extends>
   <header>lineedittel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditNome</tabstop>
  <tabstop>lineEditEmail</tabstop>
  <tabstop>lineEditTel</tabstop>
  <tabstop>comboBoxTipo</tabstop>
  <tabstop>radioButtonAtivo</tabstop>
  <tabstop>radioButtonInativo</tabstop>
  <tabstop>lineEditBanco</tabstop>
  <tabstop>lineEditAgencia</tabstop>
  <tabstop>lineEditCC</tabstop>
  <tabstop>lineEditCP</tabstop>
  <tabstop>pushButtonCadastrar</tabstop>
  <tabstop>pushButtonAtualizar</tabstop>
  <tabstop>pushButtonBuscar</tabstop>
  <tabstop>pushButtonNovoCad</tabstop>
  <tabstop>pushButtonRemover</tabstop>
  <tabstop>pushButtonCancelar</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
